src/styles/primeng-theme/components/_tooltip.scss

```diff
@@ -0,0 +1,12 @@
+.ui-tooltip {
+    &.dot-nav__tooltip {
+        min-width: 100px;
+        max-width: 150px;
```

Comment:
Can't have this, we can't know the width of this, specially in other lang and since this is user entered.

Reply:
it should wrap if it is longer than the max-width, right?

Reply:
![image](https://user-images.githubusercontent.com/31667212/71107735-4af39380-2187-11ea-8a05-b212dd615683.png)
it will wrap.